test/docs/errors/resourceObjectFields/invalidApiDocumentAttributesTypeNotRequired.js

```diff
@@ -0,0 +1,1716 @@
+/* eslint-env mocha */
+/* eslint-disable quotes */
+const invalidApiDocumentAttributesTypeNotRequired ={
```

Comment:
This line is generating linting errors. It can be resolved by running the following command:

`eslint **/*.js /test`